source/assets/public/index.css

```diff
@@ -322,7 +322,7 @@ strong {
 
 /*BODY TEXT*/
 .body-inner-wrapper {
-  max-width: 80%;
+  max-width: 90%;
```

Comment:
This makes things work better at mobile size but makes things look a little weirder at big size? I think we should actually redo how this wrapper works